e2e/tests-dfx/identity.bash

```diff
@@ -160,3 +160,13 @@ teardown() {
     assert_command dfx canister call --output idl e2e_project_frontend retrieve '("B")'
     assert_eq '(blob "hello")'
 }
+
+@test "using an unencrypted identity on mainnet provokes a warning" {
+    assert_command dfx ledger balance --network ic
+    assert_match WARN "$stderr"
```

Comment:
Please include some part of the expected warning message.  This would pass if there were any warning, not just the one added in this PR.

This suggestion specifies the whole warning message, but feel to replace parts of it with `.*` as long as assertion identifies the expected warning message.
```suggestion
    assert_match "WARN: The default identity is not stored securely. Do not use it to control a lot of cycles/ICP. Create a new identity with \`dfx identity create\` and use it in mainnet-facing commands with the \`--identity\` flag" "$stderr"
```